WooCommerce/Classes/ViewRelated/MainTabBarController.swift

```diff
@@ -382,6 +382,17 @@ extension MainTabBarController {
         ServiceLocator.stores.dispatch(action)
     }
 
+    static func switchStoreIfNeededAndPresentNotificationDetails(notification: WooCommerce.PushNotification) {
+        guard let note = notification.note else {
+            presentNotificationDetails(for: notification.noteID)
+            return
+        }
+        let siteID = Int64(note.meta.identifier(forKey: .site) ?? Int.min)
```

Comment:
Should we exist if we don't find `note.meta.identifier(forKey: .site)` as a value?

Reply:
I basically copied and modified a bit logic from `presentNotificationDetails(for noteID: Int64)` which had this line `let siteID = Int64(note.meta.identifier(forKey: .site) ?? Int.min)`. What do you think, should we return if there is no `siteID`?

Reply:
I think so, basically as the same guard as above!